packages/charts/src/vaadin-chart.js

```diff
@@ -178,7 +179,9 @@ class Chart extends ChartMixin(ThemableMixin(ElementMixin(PolylitMixin(LumoInjec
   /** @protected */
   render() {
     return html`
-      <div id="chart"></div>
+      <div style="height: 100%;">
```

Comment:
Is this element needed for charts height tests to pass? Also, should we modify logic in `__updateStyles()` as it makes chart itself a flex container and this extra element is likely to affect it?

Reply:
I updated the method. There are some tests that should've failed in the `chart-element.test.js` file. They check for resizing `vaadin-chart` elements in a flex container.